ruby/vampires.rb

```diff
@@ -1,8 +1,4 @@
-
-likes_garlic_bread = true
-wants_insurance = true
 current_year = 2017
-name = true
 
 puts "How many employees do you need to process?"
 num_of_emp = gets.chomp
```

Comment:
Make sure that you are indenting everything nested under your until for readability.  I use https://github.com/airbnb/ruby as a style guide when Im wondering how to format my code.